scripts/stress-test/knex-stress-test.js

```diff
@@ -24,6 +24,18 @@ const mysql = Knex({
   pool: { max: 50 }
 });
 
+const mssql = Knex({
+  client: 'mssql',
+  connection: {
+    port: 21433,
+    user: "sa",
+    password: "S0meVeryHardPassword",
+    server: "localhost",
+    requestTimeout: 500
+  },
+  pool: { max: 50 }
+});
+
 /* TODO: figure out how to nicely install oracledb node driver on osx
 const mysql = Knex({
   client: 'oracledb',
```

Comment:
btw, why mysql is using oracledb client?..

Reply:
just some initial copy-paste for writing oracledb test after mssql is all done.